pkg/kapp/diff/managed_fields_resource.go

```diff
@@ -0,0 +1,39 @@
+// Copyright 2020 VMware, Inc.
+// SPDX-License-Identifier: Apache-2.0
+
+package diff
```

Comment:
lets move this class into pkg/kapp/cmd/app package since it's only used by inspect command.

Reply:
done